styles/desktop-styles.css

```diff
@@ -0,0 +1,400 @@
+@media screen and (min-width: 768px) {
+  .header {
+    display: none;
+  }
+
+  .nav-desktop {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-image: url(../assets/header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    max-width: 100%;
+    max-height: 1490px;
+    width: 100%;
+    height: 70vw;
+    margin-bottom: 30px;
+  }
+
+  .text-container {
+    margin-top: 3.3rem;
+    max-width: 100%;
+    width: 56.9vw;
+  }
+
+  .bottom-arrow {
+    margin: 3em 0 5em 0;
+    padding-bottom: 10vw;
+  }
+
+  .indicator {
+    display: none;
+  }
+
+  .works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 85%;
+    height: 100%;
+    max-width: 1180px;
+  }
+
+  .works h2 {
+    font-size: 28px;
+  }
+
+  .text-container .intro {
+    font-size: 3vw;
+    font-weight: 800;
+  }
+
+  .project-grid {
+    display: grid;
+    align-self: center;
+    justify-self: center;
+    grid-template-columns: repeat(3, minmax(210px, 369px));
+    grid-template-rows: repeat(2, 1fr);
+    border-radius: 8px;
+    padding: 20px;
+    row-gap: 15px;
+    column-gap: 10px;
+    height: 100%;
+
+    /* width: 100%; */
+  }
+
+  .project-container {
+    border-radius: 8px;
+    background-color: #ebf0ee;
+    border: 1px solid #c1c7d0;
+    aspect-ratio: 3/2;
+    max-width: 369px;
+    max-height: 474px;
+  }
```

Comment:
- [ ] Please increase the padding at the bottom of your project cards to create more space between the see project button and the card border. Right now, the button is too close to the edge of the card.

- [ ] Please add some margin or padding around the tech stack list to prevent it from touching the border. Also, please add a bit more space between each tech stack list item and make the corners a bit more round to make them look more spacious.

Figma| Your's
---|---
<img width="201" alt="Screenshot 2023-07-10 at 3 04 49 PM" src="https://github.com/BHObserver/my-portfolio/assets/44798044/58c58680-683b-43f2-884f-d68826f459fe">|<img width="201" alt="Screenshot 2023-07-10 at 3 05 02 PM" src="https://github.com/BHObserver/my-portfolio/assets/44798044/f8dc877e-cd47-4578-8b60-5d63273fdf54">